test/common/testlib.py

```diff
@@ -1479,7 +1479,7 @@ def nonDestructiveSetup(self):
         self.addCleanup(m.execute, "find /var/lib/systemd/coredump -type f -delete")
 
         # temporary directory in the VM
-        self.addCleanup(m.execute, "if [ -d {0} ]; then findmnt --list --noheadings --output TARGET | grep ^{0} | xargs -r umount && rm -r {0}; fi".format(self.vm_tmpdir))
+        self.addCleanup(m.execute, "if [ -d {0} ]; then findmnt --list --noheadings --output TARGET | grep ^{0} | xargs -r umount; rm -r {0}; fi".format(self.vm_tmpdir))
```

Comment:
What do you think about explicitly using bash and also setting `pipefail`?

Reply:
Good idea!

Reply:
Ah no -- this cleans up any bind mounts from within vm_tmpdir, but most tests wouldn't have any. I.e. the `grep` is expected to fail, that's also why xargs has to use the -r option (--no-run-if-empty).